Language/Haskell/GhcMod/Cradle.hs

```diff
@@ -87,7 +95,10 @@ stackCradle wdir = do
 
     -- If dist/setup-config already exists the user probably wants to use cabal
     -- rather than stack, or maybe that's just me ;)
-    whenM (liftIO $ doesFileExist $ setupConfigPath "dist") $ mzero
+    whenM (liftIO $ doesFileExist $ setupConfigPath "dist") $ do
+                      gmLog GmWarning "" $ (text
```

Comment:
Those brackets are redundant, while you're at it please keep the string on the same line.